src/core/reionization.c

```diff
@@ -104,12 +104,12 @@ void update_galaxy_fesc_vals(galaxy_t* gal, double new_stars, int snapshot)
 #endif
       }
       break;
-    case 6: // specific star formation rate (1 / Myr)
+    case 6: // specific star formation rate (10/ Gyr)
       if ((gal->Sfr > 0.0) && (gal->StellarMass > 0.0)) {
-        fesc *= pow(gal->Sfr / gal->StellarMass / run_globals.units.UnitTime_in_s * SEC_PER_MEGAYEAR,
+        fesc *= pow(gal->Sfr / gal->StellarMass / run_globals.units.UnitTime_in_s * SEC_PER_MEGAYEAR / 100,
```

Comment:
That makes sense, just out of curiosity why are we doing this? Is it to match some particular formalism?

Reply:
i updated this based on Rosdahl MNRAS 515, 2386–2414 (2022)